src/components/Categories.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './Categories.css';
+
+const Categories = () => (
+  <div className="categories-container">
+    <p>Soon be available...</p>
+  </div>
```

Comment:
- [ ] Kindly change your categories component to display a button "Check status" only. This is a project requirement.